carball/analysis/stats/tendencies/speed_tendencies.py

```diff
@@ -30,12 +30,7 @@ def calculate_speed_for_player(cls, player: Player, proto_game: game_pb2.Game, d
 
         player.stats.averages.average_speed = average_speed
 
-        fast_frames = speed >= 22000
-        player.stats.speed.time_at_super_sonic = return_time_by_truthy_data(data_frame, fast_frames).delta
-
-        slow_frames = speed <= 7000
-        player.stats.speed.time_at_slow_speed = return_time_by_truthy_data(data_frame, slow_frames).delta
-
-        boost_frames = speed > 14100
-        player.stats.speed.time_at_boost_speed = return_time_by_truthy_data(data_frame, boost_frames).delta
+        player.stats.speed.time_at_super_sonic = sum_deltas_by_truthy_data(data_frame, speed >= 22000)
```

Comment:
Minor detail but this should probably be ordered (low to high)

Also I'm wondering if there's anywhere we should collate our arbitrarily set values